### ISSUE TITLE
I/O

### ISSUE THREAD

**Issue by profxj** [rcooke-ast/ALIS#1]
Would it be ok to generate an I/O branch to enable reading in 
other data formats (FITS, hdf5)?

**Comment by profxj** [comment]
I should add that I'm likely to use linetools (as an optional import)

**Comment by rcooke-ast** [comment]
Of course, and I had intended to include linetools as a dependency eventually anyway (for linelists). I also want to contribute some ideas I have to linetools (again... eventually).

**Comment by profxj** [comment]
Will generate a branch shortly.

**Comment by rcooke-ast** [comment]
For the record, I've added linetools dependency (the LSF function is now used). More will follow.
